object-manipulation-3/main.js

```diff
@@ -0,0 +1,90 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var deck = [];
+var rank = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K'];
+var suit = ['♧', '♢', '♡', '♤'];
+
+for (var i = 0; i < suit.length; i++) {
+  for (var j = 0; j < rank.length; j++) {
+    deck.push({ suit: suit[i], rank: rank[j] });
+  }
+}
+
+var shuffledDeck = _.shuffle(deck);
+
+function drawNewHand(players) {
+  var hand = [];
+  for (var i = 0; i < players.length; i++) {
+    hand.push({ name: players[i], hand: [], score: 0 });
+  }
+  return hand;
+}
+
+function passOutCards(hand, numCardsPerHand) {
```

Comment:
The `hand` parameter name here is confusing, because in your code, now we have `hand[k].hand. Why would `hand` hold its own `hand` property? Is the meant to be the player object? If so, you should rename the parameter to `player`